client/processes.go

```diff
@@ -148,13 +149,14 @@ func copyWithExit(w io.Writer, r io.Reader, ch chan int) {
 		}
 
 		if !isTerminalRaw {
```

Comment:
Could we remove this flag and check if we just set it to raw from the beginning?